### ISSUE TITLE
pfmon: fingerbank update job breaks DB cluster sync

### ISSUE THREAD

**Issue by dwlfrth** [inverse-inc/packetfence#2514]
**tldr;** Make `127.0.0.1` as the default DB host for:
- `pf.conf`
- `pfconfig.conf`
- `fingerbank.conf`

First occurence of the issue appeared when using Fingerbank DB update while having an active-active architecture with a galera cluster DB backend.

The Fingerbank DB update job is managed by `pfmon` and notify all members of the cluster to update themselves; including the DB...
When the issue occurs, `fingerbank.conf` was not configured to use `127.0.0.1` as the DB backend (for an unknown reason). `pfmon` was starting the update job on each of the cluster members and each of them was updating their local instance of the database, which was leading to split-brain and awful lock of tables followed by processes (connections) pilling-up due to none of them being able to finish (`query end` status due to incapacity of flushing the tx). Once that state is reached, the only way to go back is to force a new DB cluster.

Since there is apparently no way of restricting write access through local instance, avoiding mis-configuration by enforcing some defaults is probably our best way.

Action items:
- change the default DB from `localhost` to `127.0.0.1` in `pf.conf`, `pfconfig.conf`, `fingerbank.conf`;
- add a checkup script to make sure that latter settings is configured;
- create a monitoring script;
- modify fingerbank db update procedure to update only on the "master" instance when in cluster;

**Comment by dwlfrth** [comment]
Due to the impact that situation could have, I marked it as "Critical"

**Comment by dwlfrth** [comment]
I've set milestone to be 7.3 since there is no "easy way" to back port this type of fix

**Comment by fdurand** [comment]
It looks that it break the db even if we set 127.0.0.1.
Need to be fixed as soon as possible.
